pkg/sql/submitter.go

```diff
@@ -31,15 +31,25 @@ import (
 	"sqlflow.org/sqlflow/pkg/sql/ir"
 )
 
-var envSubmitter = os.Getenv("SQLFLOW_submitter")
-
 var submitterRegistry = map[string](Submitter){
 	"default": &defaultSubmitter{},
 	// TODO(typhoonzero): add submitters like alps, elasticdl
 }
 
+// SubmitterRegister registes a submitter
+func SubmitterRegister(name string, submitter Submitter) {
```

Comment:
Use a static map and remove `SubmitterRegister`, `pai_submitter.go:init()` should be simpler I think.

Reply:
It just used to be consistent with the current implementation suck as `pai_submitter.go`, it used `init()` to register the submitter.   I would clean up the submitters in #1610 .